addon/services/unified-event-handler.js

```diff
@@ -217,18 +219,25 @@ export default Ember.Service.extend(Ember.Evented, {
     return targetHandlers && targetHandlers[eventName] || undefined;
   },
 
+  /**
+   * Creates a function that is used as a handler when throttling events. This makes it possible
+   * for the same throttler function to be used for the same event name
+   * @private
+   * @param {String} emberEventName - The ember event to trigger once throttling completes
+   * @return {function}
+   */
+  _generateEventThrottler(emberEventName) {
+    return () => this.trigger(emberEventName);
+  },
+
   /**
    * Triggers a given Ember event at a throttled rate
-   * @param {String} eventName
+   * @private
+   * @param {Function} throttledEventCallback - A method that will be called at a throttled rate
    * @return {Void}
    */
-  triggerEvent(eventName) {
-    const throttleId = Ember.run.throttle(this, () => {
-      let index = this._throttledEventTimers.indexOf(throttleId);
-      this._throttledEventTimers.splice(index, 1);
-      this.trigger(eventName);
-    }, EVENT_INTERVAL);
-
+  _runThrottler(throttledEventCallback) {
```

Comment:
`_runThrottler` reads strangely, `_runThrottle` might be better

Reply:
Fixed.